docker/test/integration/minifi/__init__.py

```diff
@@ -413,7 +426,7 @@ def __init__(self,
         self.controller_services = controller_services
 
         self.schedule = {
-            'scheduling strategy': 'EVENT_DRIVEN',
+            'scheduling strategy': 'TIMER_DRIVEN',
```

Comment:
👍 by default, but longer term it would be nice to make it configurable. Sources should definitely be timer driven, in case of consumers (like publishkafka), it's not that straight forward.

Reply:
Shall I try to make it configurable in this PR or have a different jira ticket for this issue for the future?

Reply:
A follow-up is perfectly fine.